src/internals/StreamHttpReply.hpp

```diff
@@ -37,7 +39,9 @@ class AbstractStreamHttpReply
    constexpr static const char* CONTENT_TYPE_APPLICATION_JSON PROGMEM = "application/json";
 
 private:
-
+    String m_headerNames[MAX_HEADERS];
```

Comment:
If you don't mind, I think I will rewrite it to become an array of HttpField objects.

Reply:
Of course I don't mind. Your C++ is probably better than mine :).